docs/operating-eck/licensing.asciidoc

```diff
@@ -48,9 +48,9 @@ At the end of the trial period, the Platinum and Enterprise features operate in
 [float]
 [id="{p}-add-license"]
 == Add a license
-If you have a valid Enterprise subscription or a trial license extension, you will receive a license as a JSON file.
+If you have a valid Enterprise subscription or a trial license extension, you will receive a license as a JSON file. The JSON file contains the cluster-level Enterprise licenses which enables ECK features, as well Platinum Stack licenses for recent and older Elasticsearch versions.
 
-NOTE: ECK will only accept Enterprise licenses. You can not apply a single Platinum or Gold cluster license to ECK.
+NOTE: After you install a license into ECK, all the Elastic Stack applications you manage with ECK have all Platinum and Enterprise features enabled. Additionally, the link:https://www.elastic.co/guide/en/elasticsearch/reference/current/get-license.html[`_license`] API for the individual Elasticsearch clusters will report as a Platinum license, and the command `kubectl -n elastic-system get configmap elastic-licensing -o yaml` will return as Enterprise. Applications created before you installed the license are upgraded to Platinum or Enterprise features without interruption of service after a short delay.
```

Comment:
"will report as a Platinum license" and "command `kubectl -n elastic-system get configmap elastic-licensing -o yaml` will return as Enterprise" sounds a bit off to me but I am no native speaker. I think what we want to say here is that the API reports that the cluster is running under a Platinum license (which is technically not 100% correct, because if you use the right query parameters it will actually report an Enterprise license). As for the second sentence about the `kubectl` command: the command returns a JSON document that among other things contains the current license level of the ECK operator. Not sure how to express this better though.